src/Monolog/Handler/SlackHandler.php

```diff
@@ -260,36 +151,40 @@ protected function write(array $record)
      *
      * @param  int    $level
      * @return string
+     * @deprecated Use underlying SlackRecord instead
      */
     protected function getAttachmentColor($level)
     {
-        switch (true) {
-            case $level >= Logger::ERROR:
-                return 'danger';
-            case $level >= Logger::WARNING:
-                return 'warning';
-            case $level >= Logger::INFO:
-                return 'good';
-            default:
-                return '#e3e4e6';
-        }
+        trigger_error(
+            'SlackHandler::getAttachmentColor() is deprecated. Use underlying SlackRecord instead.',
+            E_USER_DEPRECATED
+        );
+
+        return $this->slackRecord->getAttachmentColor($level);
```

Comment:
Just a thought. What if someone extended `SlackHanlder` just to make colors mapping different? He'll get default colors mapping after update, right? Looks like we're breaking backwards compatibility here. Same happens with `stringify`.